Doxyfile

```diff
@@ -833,6 +833,8 @@ WARN_LOGFILE           =
 
 INPUT                  = src/Hal \
                          src/Init \
+                         src/hwas \
```

Comment:
Directory name should be capitalized, could you fix it in one go?

Reply:
I am not sure. This is generated code, I don't know if changing the folder name won't affect the build of the code that depends on hwas. @Lurkerpas can we change the name of this folder.

Reply:
@Glamhoth -> yes, this is generated code. Let's keep it as is for now.

Reply:
ack